beetsplug/bandcamp/search.py

```diff
@@ -94,21 +95,15 @@ def parse_and_sort_results(html: str, **kwargs: str) -> List[JSONDict]:
     return [{"index": i + 1, **r} for i, r in enumerate(results)]
 
 
-def get_url(url: str) -> str:
-    response = requests.get(url)
-    response.raise_for_status()
-    return unescape(response.text)
-
-
 def search_bandcamp(
     query: str = "",
     search_type: str = "",
     page: int = 1,
-    get: Callable[[str], str] = get_url,
+    get: Callable[[str], str] = http_get_text,
```

Comment:
I've been meaning to dedupe those two implementations of fetching the url data - thanks for doing this!